VMBackup/main/workloadPatch/WorkloadPatch.py

```diff
@@ -381,6 +387,36 @@ def confParser(self):
             if(self.name != None):
                 self.error_details.append(ErrorDetail(CommonVariables.FailedWorkloadConfParsingError, "exception in workloadconfig parsing"))
     
+    def updateScriptsFolder(self):
+        self.logger.log("WorkloadPatch: Creating temporary scripts folder")
+        originalScriptsFolder = os.path.join(os.getcwd(), "main/workloadPatch/"+self.scriptpath)
+        newScriptsFolder = os.path.join(self.script_folder, self.scriptpath)
+        
+        if (os.path.exists(self.script_folder) == False):
+            self.logger.log("WorkloadPatch: Script folder directory path not found..creating")
+            os.makedirs(self.script_folder)
+            
+        if (os.path.exists(newScriptsFolder)):
+            self.logger.log("WorkloadPatch: Existing temporary scripts folder found..removing")
+            self.removeScriptsFolder()
+            
+        copyProcess = subprocess.Popen(['cp','-ar',originalScriptsFolder,self.script_folder])
+        copyProcess.wait()
+        changeOwnerProcess = subprocess.Popen(['chown','-R',self.linux_user,newScriptsFolder], stdout=subprocess.PIPE)
+        changeOwnerProcess.wait()
+        permissionProcess = subprocess.Popen(['chmod','-R','700',newScriptsFolder], stdout=subprocess.PIPE)
```

Comment:
check for read and execute permission only

Reply:
Works with only read and execute, modified as per.